_data/meltano/extractors/tap-gainsightpx/Widen.yml

```
@@ -0,0 +1,61 @@
+name: tap-gainsight
+description: User tracking in web apps
+label: GainsightPX
+namespace: tap_gainsightpx
+logo_url: /assets/logos/extractors/gainsightpx.png
+variant: widen
+pip_url: git+https://github.com/Widen/tap-gainsightpx.git
+repo: https://github.com/Widen/tap-gainsightpx
+capabilities:
+- catalog
+- state
+- discover
+- about
+- stream-maps
+- schema-flattening
+settings_group_validation:
+- api_url
```

```suggestion
- - api_url
  - client_id
  - client_secret
  - refresh_token
  - start_date
```

@jlloyd-widen this key is a list of lists so it needs to be nested one more level deep.